### PR TITLE
Feat/add description list component

### PR DESCRIPTION
**Pull request by deannabosschert**
## Description
- Added the Description List component (+ Description item)
- Added the Assessment Elements component
- Clean the data that comes with it
- Rendered and styled those components in event detailpage

## Testing
`npm run dev` then go to http://localhost:3000/events/golden-dot-awards-2020/ or just check out the preview below.

### CODE REVIEW

File: src/site/_includes/components/description_list/description_list.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+.description_item {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: flex-start;
+  width: 100%;
+  margin-right: $spacing-medium;
+
```
Comment (gijslaarman) [inline src/site/_includes/components/description_list/description_list.scss:16]: Add: align-content: flex-start. Makes the text align to the top of the element.

File: src/site/_includes/components/description_list/description_list.scss [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+.description_item_icon-container {
+  display: flex;
+  align-self: center;
+  width: 25%;
+  max-width: 2.8rem;
+  padding: $spacing-tiny;
+}
+
+.description_item_icon {
```
Comment (gijslaarman) [inline src/site/_includes/components/description_list/description_list.scss:50]: Add: height to a specific number, perhaps $spacing-large e.g.

remove: max-height & max-width.

Comment (deannabosschert) [reply src/site/_includes/components/description_list/description_list.scss:51]: Done!

File: src/site/_includes/components/description_list/description_list.scss (hunk 1 above)
Comment (deannabosschert) [reply src/site/_includes/components/description_list/description_list.scss:17]: Done!

**Comment by deannabosschert** [comment]
Changes applied; https://cmd-digital-playground-aobng0iv9.vercel.app/events/golden-dot-awards-2020

**Pull request merged**